Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM python:3.10-slim
+
+WORKDIR /app
+
+COPY . .
+
+RUN pip install --no-cache-dir -r requirements.txt
+
+ARG ARG_AZURE_API_ENDPOINT
+ARG ARG_AZURE_API_KEY
+ARG ARG_AZURE_OPENAI_API_ENDPOINT
+ARG ARG_AZURE_OPENAI_API_KEY
+ARG ARG_PROMPT_PATH
+ARG ARG_UPLOAD_PATH
```

Comment:
Add this [envar ](https://github.com/ai-cfia/fertiscan-backend/blob/4c5a11fe51e0703566989be547c04e240057962d/.env.template#L12) to the list.

Reply:
Good I will also update the README.md